v2/pkg/common/constants.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package common
+
+// =================== Config variable constants ===================
+
+const (
+	// #nosec
+	ClientSecretVar      = "AZURE_CLIENT_SECRET"
+	SubscriptionIDVar    = "AZURE_SUBSCRIPTION_ID"
+	TenantIDVar          = "AZURE_TENANT_ID"
+	ClientIDVar          = "AZURE_CLIENT_ID"
+	ClientCertificateVar = "AZURE_CLIENT_CERTIFICATE"
+	// #nosec
+	ClientCertificatePasswordVar = "AZURE_CLIENT_CERTIFICATE_PASSWORD"
+	TargetNamespacesVar          = "AZURE_TARGET_NAMESPACES"
+	OperatorModeVar              = "AZURE_OPERATOR_MODE"
+	SyncPeriodVar                = "AZURE_SYNC_PERIOD"
+	ResourceManagerEndpointVar   = "AZURE_RESOURCE_MANAGER_ENDPOINT"
+	ResourceManagerAudienceVar   = "AZURE_RESOURCE_MANAGER_AUDIENCE"
+	AzureAuthorityHostVar        = "AZURE_AUTHORITY_HOST"
+	PodNamespaceVar              = "POD_NAMESPACE"
+	UseWorkloadIdentityAuth      = "USE_WORKLOAD_IDENTITY_AUTH"
+	// #nosec
+	FederatedTokenFilePath = "/var/run/secrets/tokens/azure-identity"
+)
+
+// =================== MultiTenant Constants ===================
+
+const (
+	PerResourceSecretAnnotation = "serviceoperator.azure.com/credential-from"
+)
+
+// =================== Reconcile Policy Constants ===================
+
+// ReconcilePolicyAnnotation describes the reconcile policy for the resource in question.
+// A reconcile policy describes what action (if any) the operator is allowed to take when
+// reconciling the resource.
+// If no reconcile policy is specified, the default is "run"
+const ReconcilePolicyAnnotation = "serviceoperator.azure.com/reconcile-policy"
+
+const (
+	// ReconcilePolicyManage instructs the operator to manage the resource in question.
```

Comment:
Should we keep these comments here, in `reconcile_policy.go` or both places?

Reply:
Have to do it this way to avoid cyclic imports. Also, would be good to expose these values to external users with comments to have an idea of what the value does.

Reply:
I think we move the `ReconcilePolicy` type here too, retaining type safety for clients who want it (and the others can cast it if they need to).